topic_folders/lesson_development/lesson_pilots.md

```diff
@@ -0,0 +1,125 @@
+to assess the design and flow of the lesson.
+This [template post-pilot workshop survey](https://docs.google.com/forms/d/1OGCQBotD2nOJkc7KpFZLhFfb3EBcxEDwHz_3p48qz3U/template/preview)
+can be copied and adapted to suit the needs of your lesson,
+and shared with learners in place of the standard post-workshop survey.
+
+It is also important to gather information about the lesson
+while it is being taught.
+Check
+[the Lesson Life Cycle chapter of The Carpentries Curriculum Development Handbook](https://cdh.carpentries.org/the-lesson-life-cycle.html#field-testing-alpha-stage)
+for a list of things to take note of during the pilot workshop.
+We recommend assigning a specific person or people with responsibility
+to keep track of these points, e.g. an Instructor or Helper.
+
+### Information for Hosts
+
+#### Recruiting Instructors for Beta Pilots
+
+If you are hosting a pilot of a new official Carpentries lesson -
+a lesson developed based on prior agreement with The Carpentries,
+intended to become another lesson/curriculum offered
+in centrally-organised workshops -
+[the Curriculum Team](mailto:team@carpentries.org) will help you
+find Instructors for pilot workshops.
+
+The Carpentries is also keen to support the development and piloting
+of lessons in The Carpentries Incubator.
+If you are hosting a pilot of a lesson in the Incubator,
+we ask that you **first** try to find Instructors for pilot workshops yourself.
+Often, hosts are able to recruit certified Instructors from their local
+community
+with relevant knowledge of the lesson topic
+but in some cases this will not be possible.
+If you wish to recruit Instructors for a pilot workshop,
+try putting a call out on
+[local/regional community mailing lists](https://carpentries.topicbox.com/groups),
+any relevant channels on [The Carpentries Slack workspace](https://carpentries.org/connect/) (the lesson authors may be able to direct you to these),
+and/or [by publishing a post on our blog](https://docs.carpentries.org/topic_folders/communications/guides/submit_blog_post.html).
+Please do not post calls for Instructors to the general channel on Slack, or the discuss and instructors lists on TopicBox: any messages to recruit Instructors will be removed from those channels. If after taking these steps, you find that you need help finding Instructors for your lesson pilot, you can [contact incubator@carpentries.org](mailto:incubator@carpentries.org)
+for assistance.
+
+#### Creating a Pilot Workshop Webpage
+
+[The Carpentries workshop webpage template](https://github.com/carpentries/workshop-template)
+supports the creation of webpages for pilot workshops.
+See [the Customisation page of the template documentation](https://carpentries.github.io/workshop-template/customization/#configuration-file-_configyml)
+for details of how to configure the webpage for a pilot workshop.
+
+The Carpentries does not currently track pilot workshops in the same
```

Comment:
We do track pilots for official-track lessons. They should be reported as self-organized workshops using the standard form. (Sher and I will work together to make sure the curriculum that is being piloted gets added to the form before pilots start.)

Reply:
Thanks for catching that inaccuracy, @ErinBecker. I have corrected in a0f2a56